### DISCUSSION TITLE
8 relay board esp32 lights and blinds both usage

### DISCUSSION THREAD

**Discussion by aytaral** [arendst/Tasmota#20407] [Q&A] [answered]
Hello,
İ have 8 relay board and connect esp32 woroom. 
First four relay control lights.
İ want to use another four relays to control blinds up and down. 
Relay 1,2,3,4 connected lights. 
İ have 2 blinds. 
İ connected
Blind1 Relay 5 up, 6 down
Blind2 Relay 7 up, 8 down. 

İts possible this setup? 
And i want to use this setup in homeassistant. 
But first how i setup in tasmota? 

Thanks

**Comment by HofeBY** [comment, marked as answer]
Relay´s are ready configured !
.
https://tasmota.github.io/docs/Blinds-and-Shutters/#calibration
https://tasmota.github.io/docs/Commands/#setoptions
.
Now some Commands in Console:
SetOption80 1
ShutterMode 1
Restart 1
Interlock 5,6 7,8
ShutterRelay1 1
ShutterRelay2 2
.
//Adopt runtime for shutter1; second replace 1by2:
ShutterCloseDuration1 16 // runtime in seconds to close shutter
ShutterOpenDuration1 18 // runtime in seconds to open shutter, alway´s 2secs more than next value
ShutterOpen1 // open shutter "power1 1"
ShutterClose1 // close shutter alternativ "power2 1"
ShutterPosition1 100 // Position run to 100=open 0=closed 60=parttime closed
.
Now the next one:
ShutterCloseDuration2 16 // runtime in seconds to close shutter
ShutterOpenDuration2 18 // runtime in seconds to open shutter, alway´s 2secs more than next value
ShutterOpen2 // open shutter "power2 1"
ShutterClose2 // close shutter alternativ "power2 0"
ShutterPosition2 100 // Position run to 100=open 0=closed 60=parttime closed
.
// Example to remote start Shutter 5 for up, 6 for down etc.:
http://IPv4OfDevice/cm?cmnd=Power5%20On
http://IPv4OfDevice/cm?user=admin&password=joker&cmnd=Power5%20On
.
Hope everything is right.
HofeBY from Bavaria

**Reply by aytaral** [reply]
Thanks for your detailed information. i try today.
Is it necessary to make these settings when the shutter is physically fully closed?
To determine the initial position?

**Comment by HofeBY** [comment]
I use Shuttermotors with physical adjusted endstops.
One regulater for upper Endstop and another for bottom endstop.
something like this one
https://www.ebay.de/itm/231136181153?hash=item35d0ca3ba1:g:P~UAAOSwzJ5XZoce&amdata=enc%3AAQAIAAAA4NpsJk6icv1ae6qemXbs3WaILULBdinz5pUgQ%2F6K2LrtqOICoW9MOjwdO4JqTdCULFM0T%2F0Q2RvT2sQc0kn%2FFiNHzr769pXLyASOgqbHnj%2Bw%2BP5i2STYRdwaoICFkyA3ka0AYF1DlKsT%2BbV5XFh%2FhafITrgMQ%2Fpxotxc2judHY9AcRTQ%2BsPc04MotTPRY4xCzQQUnjvcmAfNGvVbZW9GCCpmvcZ3TnmRpELY7KkVXEMLBYC6iLU4Mp52UZ%2F1sOKDmOdEpmPoh4r8FV%2Fn%2BbZzBJkqhLYFg1ooeae%2FQ5JA4Si1%7Ctkp%3ABk9SR8S49K-dYw

Greetings from Bavaria
